aroma/utils.py

```diff
@@ -327,9 +327,8 @@ def classification(features_df, out_dir):
     )
 
     # Put the indices of motion-classified ICs in a text file (starting with 1)
-    motion_ICs = features_df[
-        "classification", features_df["classification"] == "rejected"
-    ].index.values
+    motion_ICs = features_df["classification"][features_df["classification"] == "rejected"].index
+    motion_ICs = motion_ICs.values
```

Comment:
Why this change?

Reply:
That was failing too after correcting the `x.T` error

Reply:
Oh, sorry I see it now. I didn't notice the `][` being changed to `, ` and misinterpreted it. I thought it was just a cosmetic change.